.ci.yaml

```diff
@@ -1969,7 +1969,8 @@ targets:
 
   - name: Linux_android flutter_gallery__transition_perf
     recipe: devicelab/devicelab_drone
-    presubmit: false
+    presubmit: true
```

Comment:
Let's keep `presubmit: false`.

Reply:
Gotcha. It should have been false. I'll go ahead and change it.